public/blog/testrpc-is-now-ganache-cli.md

```diff
@@ -0,0 +1,25 @@
+# TestRPC is now Ganache CLI
+
+Have you been looking for the TestRPC recently and haven't been able to find anything about it?
+
+Don't worry, it's still here! It's just now known as **Ganache CLI**. You can get it here:
+
+```shell
+npm install -g ganache-cli
+```
+
+## A little history
+
+Prior to the existence of the TestRPC, one would have to deploy to a test network like Ropsten in order to see how your contract operates. The TestRPC solved that issue by providing a local blockchain where you could test your smart contracts without even needing to wait for your transactions to be mined into a block. 
+
+While initially under the auspices of the [EthereumJS project](https://github.com/ethereumjs/), it was designed by the Truffle team from the beginning, and today we remain its primary architect.
+
+So bringing the TestRPC under the wing of the Truffle project just made sense.
+
+Separately, we have been hard at work on a new, graphical blockchain application called [Ganache](/ganache). This provided many of the same features as the TestRPC, but in a more user-friendly graphical format. In fact, Ganache used the TestRPC under the hood so much that the two tools are really versions of the same thing.
```

Comment:
This paragraph talks about Ganache and the TestRPC as if they're two different things. I'd like to keep the branding the same, as if they're two different flavors of the same thing. I'd also like to reference TestRPC being generally a bad name in that A) it's way more powerful than the name implies, B) is not just for tests, and C) may even get rid of the JSON RPC interface eventually! I'd do something like this:

> Additionally, we realized over time that the TestRPC was a poor name. The TestRPC quickly became more powerful than simply a blockchain environment used for testing. To stay with the sweet Truffle brand, we decided to name it Ganache, as Ganache is (sometimes) the core of your favorite chocolate truffle. It's also a great topping. 
> 
> With this name change we also released a fully-interactive, desktop version of Ganache which you can find here (LINK ME)